packages/standard-minifiers-css/plugin/minify-css.js

```diff
@@ -14,17 +14,19 @@ CssToolsMinifier.prototype.processFilesForBundle = function (files, options) {
 
   if (! files.length) return;
 
-  var merged = mergeCss(files);
-
+  // Don't merge or minify anything for development
   if (mode === 'development') {
-    files[0].addStylesheet({
-      data: merged.code,
-      sourceMap: merged.sourceMap,
-      path: 'merged-stylesheets.css'
+    files.forEach(function (file) {
+      file.addStylesheet({
+        data: file.getContentsAsBuffer(),
+        sourceMap: file.getSourceMap(),
+        path: file.getPathInBundle()
```

Comment:
This seems extremely related to your open pull request https://github.com/meteor/meteor/pull/5841

I will look into this when I have a chance. I think this is a pretty worthwhile change, it could decrease many people's client reload times by >2s (and I'm on a really fast computer)!